src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -580,6 +592,12 @@ private JsonObject findAccountForFeeFineAction(JsonObject feeFineAction) {
     return accountsClient.get(UUID.fromString(feeFineAction.getString("accountId"))).getJson();
   }
 
+  private JsonObject modifyFields(JsonObject feeFineAction, String field, String otherField) {
```

Comment:
How do you think to rename the method to "scaleFileds"? and rename params, fieldOne and fieldTwo?

Reply:
make sense